yt/docs/en/_includes/admin-guide/install-ytsaurus.md

```diff
@@ -289,7 +289,8 @@ Events:
 Follow all required steps to [start the {{product-name}} cluster](#starting-cluster). Then install the chart:
 
 ```
```

Comment:
Here and below, to highlight the syntax in the code snippets, you can specify the `bash` language in the initial line:

\`\`\`bash
...
\`\`\`